blackjack/deck.js

```diff
@@ -0,0 +1,54 @@
+class Card {
+    constructor(suit, value) {
+        this.suit = suit;
+        this.value = value;
+    }
+}
+
+class Deck {
```

Comment:
deck
methods:
- draw()
- shuffle()
- createDeck()

attributes:
- cards / deck

hand shouldn't be in the deck